services/ui/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM httpd
+COPY httpd.conf /usr/local/apatche2/conf/

```

Comment:
Should be `apache`, not `apatche` in both places, right?

Reply:
fixed